infrastructure/pipeline_stack/unpublisher_lambda.tf

```
@@ -0,0 +1,69 @@
+locals {
+  unpublisher_lambda_name = "content-unpublisher-${var.pipeline_date}"
+}
+
+module "unpublisher_lambda" {
+  source = "git@github.com:wellcomecollection/terraform-aws-lambda?ref=v1.2.0"
+
+  name    = local.unpublisher_lambda_name
```

Maybe add a one-line description for users in the console?

```suggestion
  name    = local.unpublisher_lambda_name
  description = "Handle any documents which are "unpublished" (deleted) in Prismic"
```